tests/follow.test.js

```diff
@@ -0,0 +1,84 @@
+import chai from 'chai';
+import chaihttp from 'chai-http';
+import server from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+import generateToken from '../src/helpers/tokens/generate.token';
+import env from '../src/configs/environments';
+import errorMessage from '../src/helpers/constants/error.messages';
```

Comment:
errorMessages makes more sense @truestbyheart

Reply:
if in my code I write  ```errorMessage.noUser``` and ```errorMessages.noUser```, for me I think the former makes more sense cause am its a single error am testing for.

Reply:
cool @truestbyheart